doc/releases/release-notes-2.3.rst

```diff
@@ -160,7 +160,14 @@ Architectures
 
 * ARC:
 
-  * <TBD>
+  * Changed to privilege stack generation for ARC MPU version 2 to avoid
+    memory waste
+  * Added MPU GAP FILLING for ARC MPU version 3
```

Comment:
Suggestion: (thanks Ioannis!)
"Enhanced runtime programming for the MPU v3 by making the gap-filling of kernel memory a user-configurable
    feature"